src/security/auth.js

```diff
@@ -1,6 +1,15 @@
+import { isExpired } from "@/helper/date.helper";
+import store from "@/security/store";
+
 class Auth {
   check() {
```

Comment:
I'm not quiet convinced to that name. I saw some approaches like that, but usually it throws error when something is wrong. So you don't use it like `if(check()) {...}` you just call this and wait for it to pass or throw error. this method is basically checking if token is still valid so I suggest something like `isTokenValid`